cmd/sourced/compose/compose.go

```diff
@@ -78,6 +78,10 @@ func getOrInstallComposeBinary() (string, error) {
 }
 
 func getOrInstallComposeContainer() (string, error) {
+	if err := dir.Prepare(); err != nil {
```

Comment:
I don't think that `Compose` should be responsible for preparing the directory. I think that the commands should already have that dir ready before using compose.

Reply:
It is not done by `Compose`, but for the helper requiring that directory.

Anyhow, preparing `~/.sourced` dir on `sourced` commands was my first attempt but discarded because then I thought you would prefer this other approach.

Ok.
So to be on the same page @se7entyse7en @smacker 
- It should be called on every `sourced` command because `~/.sourced` dir is a core thing in our app.
  - more simple, but will include `version` and `completion`
- It should be called only on `subcommands` using that dir.
  - more precise, but done in each subcommand, so it could be forgotten in new subcommands.

Reply:
I'd do it for every `sourced` command. Core things will require that, so if you're gonna the product sooner or later you're gonna need that dir. It's also easier IMO. The only downside is for users that will download `sourced` and will only use commands that actually don't require that dir. But I think that it's not very likely, and also that we shouldn't care that much.

Reply:
thanks for helping!!

done by 8f2d60f0008843f49a5afec25e4968ea598c6004